package.json

```diff
@@ -93,6 +93,7 @@
     "eslint-config-next": "13.4.9",
     "eslint-config-prettier": "8.8.0",
     "eslint-plugin-mdx": "2.1.0",
+    "eslint-plugin-no-relative-import-paths": "^1.5.2",
```

Comment:
Please read the DEPENDENCY_PINNING.md, this dependency should use an exact version, instead of “^”